django/contrib/sitemaps/management/commands/ping_google.py

```diff
@@ -7,6 +7,12 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         parser.add_argument('sitemap_url', nargs='?')
+        parser.add_argument('-d', '--site_domain', default=None)
```

Comment:
Options usually use a `-` for separators.
```suggestion
        parser.add_argument('-d', '--site-domain', default=None)
```

Reply:
I believe `default=None` isn't needed because that's the default (possibly the same with `default=False` below).